docs/commands/json.mset.md

```diff
@@ -1 +1,60 @@
-Sets or updates the JSON value of one or more keys.
+Set or updates JSON values of one or more keys (multiple) for a given tripplet `key-path-value`
+
+JSON.MSET is atomic, so all given keys are set at once or not. It is not possible for clients to see that some of the keys were updated while others are unchanged.
+
+Important, JSON is a hierarquical structure if you change a value in specific path, subsequent nested values will be affected and may be unchanged
```

Comment:
<note><b>Note:</b>
A JSON value is a hierarchical structure. If you change a value in a specific path - nested values are affected. 
</note>